fabmukunzi/posinnove-dashboard
# Feat reset password

# Reset password
## This PR is here for the process of:
- forget password request
- reset password

### Broker
  -None :)
  
  ## How to test it?
- run `git clone https://github.com/fabmukunzi/posinnove-dashboard` in your terminal
- run `git checkout feat-reset-password` in your terminal
- run `yarn install` to start the app
- got to login page
- click on forgot password
- add your email and send
- check your email link
- click on link
- add your password and confirm it
- send
